src/main/fc/config.c

```diff
@@ -429,6 +429,8 @@ static void activateConfig(void)
     activateControlRateConfig();
 
     resetAdjustmentStates();
+    
+    useRcControlsConfig();
```

Comment:
It gives false impression that function actually *updates* the mode activation conditions while it actually only processes them and caches some flags.
`updateUsedModeActivationConditionFlags()` ?